components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1755,7 +1755,7 @@ void RewardsServiceImpl::GetPublisherActivityFromUrl(
       GetDomainAndRegistry(origin.host(), INCLUDE_PRIVATE_REGISTRIES);
   std::string path = parsed_url.PathForRequest();
 #if BUILDFLAG(IPFS_ENABLED)
-  if (ipfs::IsIpfsEnabled(profile_) && parsed_url.SchemeIs(ipfs::kIPNSScheme)) {
```

Comment:
is it safe to drop this check?

Reply:
yes, that check was removed here and we decided to rework rewards code in future but keep it like this right now